src/Models/Note.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tipoff\Notes\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Note extends Model
+{
+    use HasFactory;
```

Comment:
Instead of HasFactory, we need to use the HasPackageFactory trait

Reply:
@drewroberts I can't find a package where the models don't use that trait. All of them extend the HasFactory trait as well.

Reply:
Thank you. When you get a chance, run a PR on those other packages to update them as well.